frontend/src/diagram/DiagramWebSocketContainer.tsx

```diff
@@ -455,7 +457,11 @@ export const DiagramWebSocketContainer = ({
       if (tool) {
         const { id: toolId } = tool;
         if (tool.__typename === 'CreateEdgeTool') {
-          const [diagramSourceElementId, diagramTargetElementId] = params;
+          const [diagramSourceElementId, diagramTargetElementId, sourcePosition, targetPosition] = params;
```

Comment:
Can sourcePosition really be null? because the values are required in the GraphQL API.